src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    partnerName,
+    sex,
+    isMarried,
+  } = person;
+  const MALE = 'm';
+  const WIFE = 'wife';
+  const HUSBAND = 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        { `My name is ${name}` }
+      </h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${sex === MALE ? WIFE : HUSBAND}`
```

Comment:
It is still nested condition. You need to create variable for this value - `${sex === MALE ? 'wife' : 'husband'}` and use it here